.gitignore

```diff
@@ -31,3 +31,6 @@ build/
 
 ### VS Code ###
 .vscode/
+
+### Docker ###
+.env
```

Comment:
why do you ignore env.file?

Reply:
I thought it was a good idea) I will fix it now.